src/main/java/com/stripe/model/StripeObject.java

```diff
@@ -34,12 +36,30 @@ public void setLastResponse(StripeResponse response) {
     this.lastResponse = response;
   }
 
+  /**
+   * Returns the raw JsonObject exposed by the Gson library. This can be useful to access properties
+   * that are not declared in the Stripe Java library.
+   *
```

Comment:
Would you mind also adding a note here that explains that because this exposes some of the GSON API, it should not be considered fully stable, and that it's vastly preferable to use the standard static properties on a `StripeObject`. It's possible that a future version of stripe-java could move off of GSON and this method will either go away or change significantly.